### PR TITLE
ADT-114 - Fix invalid persona RowKey

### PR DESCRIPTION
**Pull request by sfitz42**

### CODE REVIEW

File: app/storage/repos/persona.js [hunk 1]
```diff
@@ -8,7 +8,9 @@ const initialiseTable = async () => {
 }
 
 const calculateRowKey = (name, version) => {
-  const escaped = name.toLowerCase().replace(' ', '_')
+  const escaped = name.toLowerCase().replaceAll(' ', '_')
+
+  console.log(escaped)
```
Comment (sebhoron) [inline app/storage/repos/persona.js]: Should we remove console.log?

**Review by sebhoron — COMMENTED** [review]
Looks good, just left 1 comment

File: app/storage/repos/persona.js (hunk 1 above)
Comment (sfitz42) [reply app/storage/repos/persona.js]: Yep good catch :)

**Review by sebhoron — APPROVED** [review]
LGTM

**Pull request merged**